<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(246, 235, 97);</string>
  </property>
  <widget class="QWidget" name="widgetPatTrash" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>520</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEditSearchTrash">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color:  rgb(0, 51, 102);
border: 2px solid rgb(0, 51, 102);
border-radius: 5px;</string>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Search Patient</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetTrash">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>129</y>
      <width>731</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
    border: 1px solid  rgb(0, 51, 102);
    background-color: white;
    gridline-color: #dcdcdc;
}

QHeaderView::section {
    background-color: rgb(0, 51, 102);
    color: white;
    padding: 2px;
}

QTableWidget::item {
    font-size: 8px;
	padding-left: 2px;
}
</string>
    </property>
    <column>
     <property name="text">
      <string>Patient ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Last Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>First Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Contact Number</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Age</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Last Visit</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 51, 102);
color: white;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Trash</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
